main.py

```diff
@@ -351,5 +297,15 @@ def run(self):
 
 
 if __name__ == '__main__':
-    sdw = SiggyDeltaWarnings()
+    if (WEBHOOK_TYPE == 'discord'):
```

Comment:
Would prefer this come from a utility function in the WebHookAdapater along the lines of `get_webhook_adapter(WEBHOOK_TYPE)` that returns the correct adapter based on the webhook type or raises an error.